### PR TITLE
Fix #1 -- Add settings to make the field required and add a help text

### LINKED ISSUE #1: Add option to make telephone field required
Someone might need it.

### PR DESCRIPTION
**Pull request by raphaelm**
Hi,

for a customer we need a required telephone field with a certain help text, so I figured the best way would be to extend this plugin. On the way, I changed the (English) language slightly, as I think it feels more natural.

Happy for any feedback!

Raphael

### CODE REVIEW

File: pretix_telephone/views.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+from pretix.base.forms import SettingsForm
+from pretix.base.models import Event
+from pretix.control.views.event import EventSettingsFormView, EventSettingsViewMixin
+
+
+class TelephoneFieldSettingsForm(SettingsForm):
+    telephone_field_required = forms.BooleanField(
+        label=_("Require phone number"),
+        help_text=_("If this is not checked, entering a phone number is optional.."),
```
Comment (felixrindt) [inline pretix_telephone/views.py]: There are two `.` at the end of the sentence :)

**Comment by felixrindt** [comment]
Hi. Glad to see you :) This fixes #1 btw :+1:

File: pretix_telephone/views.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        required=False,
+    )
+    telephone_field_help_text = forms.CharField(
```
Comment (felixrindt) [inline pretix_telephone/views.py]: This should be an I18nField to allow for multilingual help texts.

Comment (raphaelm) [reply pretix_telephone/views.py]: Wow...this happening to me... will fix xD

**Comment by raphaelm** [comment]
Thanks for the review! I pushed a new version that should fix the issues you found.

**Comment by felixrindt** [comment]
Great! Thanks for your work! I'm sure we'll find a good use for the help text aswell :)

**Pull request merged**